transformers/ArbitrumPollingTransformer.js

```diff
@@ -47,13 +55,43 @@ const handlers = {
       optionIdInt = info.event.params.optionId.toNumber();
     }
 
+    let delegateContractAddress;
+
+    try {
+      // If we are on the L2 testnet, use the Goerli deployment of Vote Delegate Factory.
+      const vdfAddress =
+        layer2ChainMap[services.networkState.networkName.chainId];
+
+      // The provider needs to be connected to the same L1 network where the delegate contract was created.
+      const provider = new ethers.providers.JsonRpcProvider(
+        process.env.VL_CHAIN_HOST
+      );
+      const delegateFactoryContract = new ethers.Contract(
+        vdfAddress,
+        vdfAbi,
+        provider
+      );
+      delegateContractAddress = await delegateFactoryContract.delegates(
+        info.event.params.voter
```

Comment:
Does this return the zero address if it doesn't exist? Looks like it based of your code, but just making sure.